libs/shared/providers/src/prices/constants.ts

```diff
@@ -313,13 +313,13 @@ export const priceOptions: Partial<Record<SupportedTokenPrice, PriceOption>> = {
       return +formatUnits(sweth_eth, tokens.mainnet.swETH.decimals);
     },
   },
-  swETH_USD: {
-    id: 'swETH_USD',
+  'swETH:1_USD': {
+    id: 'swETH:1_USD',
     type: 'derived',
-    dependsOn: ['swETH_ETH', 'ETH_USD'],
+    dependsOn: ['swETH:1_ETH', 'ETH:1_USD'],
   },
-  WETH_ETH: {
-    id: 'WETH_ETH',
+  'WETH:1_ETH': {
```

Comment:
looks weird, mainnet could be implicit (if you don't specify it is assumed)
has me wishing for a better way

Reply:
I agree, it doesn't look good... I reverted